ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -263,6 +264,27 @@ ColumnLayout {
         }
     }
 
+    Component {
+        id: unfurledProfileLinkComponent
+        UserProfileCard {
+            id: unfurledProfileLink
+            readonly property var contact: Utils.parseContactUrl(parent.link)
+            readonly property var contactDetails: Utils.getContactDetailsAsJson(contact.publicKey)
+
+            leftTail: !root.isCurrentUser
+            userName: contact && contact.displayName ? contact.displayName : 
+                      contactDetails && contactDetails.displayName ? contactDetails.displayName : ""
+                      
+            userPublicKey: contactDetails && contactDetails.publicKey ? contactDetails.publicKey : ""
```

Comment:
... and should not display the pubkey if the user has an ENS name

Reply:
We don't display it. It's used to generate the emoji hash